libs/coin-evm/package.json

```diff
@@ -71,6 +68,9 @@
   },
   "devDependencies": {
     "@types/imurmurhash": "^0.1.4",
+    "@ledgerhq/hw-app-eth": "workspace:^",
```

Comment:
Are we sure it's only a dev dependency ? It looks like we import ledgerService here : 
https://github.com/LedgerHQ/ledger-live/blob/support/evm-documentation/libs/coin-evm/src/signOperation.ts#L9

Which looks like an object with methods :
https://github.com/LedgerHQ/ledger-live/blob/support/evm-documentation/libs/ledgerjs/packages/hw-app-eth/src/services/ledger/index.ts#L224

Reply:
Fixed by using the `clearSignTransaction` method instead from `hw-app-eth` and not relying on the `ledgerService` import.